### PR TITLE
Move code to storage module

### PR DESCRIPTION
**Pull request by SergSkol**
- Move code to "storage.js" to keep localStorage manipulations separatelly;

### CODE REVIEW

File: src/modules/ToDoList.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
+import { saveTasks } from './storage.js';
+
 export default class ToDoList {
   constructor() {
```
Comment (girma336) [inline src/modules/ToDoList.js:4]: # you should use inside constructor  more readable

File: src/modules/ToDoList.js [hunk 2]
```diff
@@ -50,17 +51,6 @@ export default class ToDoList {
...
-  }
-
-  loadTasks() {
-    this.arr = JSON.parse(localStorage.getItem(this.storageKey));
-    if (this.arr == null) {
-      this.arr = [];
-    }
-  }
-
   showTasks() {
```
Comment (girma336) [inline src/modules/ToDoList.js]: - You should use es6 arrow function

File: src/modules/storage.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+const storageKey = 'todolist';
+
+const saveTasks = (arr) => {
+  localStorage.setItem(storageKey, JSON.stringify(arr));
+};
+
+const loadTasks = () => {
+  let arr = JSON.parse(localStorage.getItem(storageKey));
+  if (arr == null) {
```
Comment (girma336) [inline src/modules/storage.js]: - You should you instead of '==' use strict equal sign '==='

File: src/modules/ToDoList.js (hunk 2 above)
Comment (SergSkol) [reply src/modules/ToDoList.js]: Thanks!

File: src/modules/storage.js (hunk 3 above)
Comment (SergSkol) [reply src/modules/storage.js]: Thanks!

**Review by girma336 — COMMENTED** [review]
- No liners error 
- keep it !!!  up noting to say

**Review by girma336 — APPROVED** [review]
Hi @SergSkol 
Congratulations!
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Status: APPROVED

**Pull request merged**